src/test/java/com/salesforce/storm/spout/dynamic/kafka/ConsumerTest.java

```diff
@@ -454,8 +461,10 @@ public void testConnectWithSinglePartitionOnTopicWithNoStateSaved() {
         // Since ConsumerStateManager has no state for partition 0, we should call seekToBeginning on that partition
         verify(mockKafkaConsumer, times(1)).seekToBeginning(eq(Lists.newArrayList(kafkaTopicPartition0)));
 
-        // Verify position was asked for
-        verify(mockKafkaConsumer, times(1)).position(kafkaTopicPartition0);
+        final ConsumerRecords<byte[], byte[]> consumerRecords = makeConsumerRecords(0, 1);
+        when(mockKafkaConsumer.poll(300)).thenReturn(consumerRecords);
+
+        consumer.nextRecord();
```

Comment:
@Crim By dropping the position() call, the first call to nextRecord() gets the state in the right spot on a reset to the beginning. This seems ok to me, yeah?  Do we really need to be persisting state before we've done a poll()?